### ISSUE TITLE
[Glance 2.1] Separate versions for 2.1 and 2.0

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-intro-wcag#16]
From @yatil https://github.com/w3c/wai-intro-wcag/pull/15#issuecomment-392605672

> Didn’t we want to get rid of version numbers? If the WCAG 2.1 stuff is clearly marked, we should be good. I would not want to have two versions of the page, and restricting it to WCAG 2.1 (by the title) is not too useful, imho.

**Comment by yatil** [comment]
Thanks, that clarifies it for me. No need to discuss, I just wanted to know what our approach is.

(Note: I did not reference this issue in #15. Shawn did change my comment.)
